scripts/cf-operator-upgrade.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+source scripts/include/setup.sh
+
+require_tools cf_operator_url kubectl helm
+
+if [ -z "${CHART:-}" ]; then
+    CHART="$(cf_operator_url)"
+fi
+
+helm upgrade cf-operator \
```

Comment:
Can we not change `cf-operator-apply.sh` to do `helm upgrade --install` instead of having a new script?

Reply:
Reverted, see previous comment